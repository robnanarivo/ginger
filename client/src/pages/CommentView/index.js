import CommentView from './CommentView';

export default CommentView;
