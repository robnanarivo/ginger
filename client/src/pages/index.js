import Login from './Login/Login';
import Signup from './Signup/Signup';
import Profile from './Profile/Profile';
import CreatePost from './CreatePost/CreatePost';
import MainView from './MainView/MainView';
import GroupView from './GroupView';
import NewGroup from './NewGroup/NewGroup';
import Chat from './Chat/Chat';
import CommentView from './CommentView';
import GroupManage from './GroupManage';

export {
  Login,
  Signup,
  MainView,
  GroupView,
  Profile,
  CreatePost,
  NewGroup,
  GroupManage,
  CommentView,
  Chat,
};
