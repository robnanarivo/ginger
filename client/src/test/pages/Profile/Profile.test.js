import React from 'react';
import renderer from 'react-test-renderer';
import { createMemoryHistory } from 'history';
import { Router } from 'react-router-dom';
// import { render, screen } from '@testing-library/react';
import { Profile } from '../../../pages';
import 'jest-styled-components';

describe('Profile unit test', () => {
  it('Profile Snapshot test', () => {
    const history = createMemoryHistory();
    const tree = renderer
      .create(
        <Router history={history}>
          <Profile isSelf />
        </Router>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  // it('Profile contains update button', () => {
  //   render(<Profile isSelf />);
  //   // contains cancel button
  //   expect(screen.getByText(/update/i)).toBeInTheDocument();
  // });
});
