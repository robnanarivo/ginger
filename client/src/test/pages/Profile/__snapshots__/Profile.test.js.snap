// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile unit test Profile Snapshot test 1`] = `
<div
  className="MuiBox-root css-19a20d"
>
  <div
    className="MuiBox-root css-487fe5"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation3 css-3su884-MuiPaper-root"
    >
      <div
        className="MuiBox-root css-16ggt3o"
      >
        <div
          className="MuiBox-root css-yuob64"
        >
          <div
            className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-hidgkg-MuiAvatar-root"
          >
            
          </div>
        </div>
        <p
          className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          
        </p>
        <span
          className="MuiTypography-root MuiTypography-caption css-qkcnm0-MuiTypography-root"
        >
          Member since
           
          January 1, 1970
        </span>
        <div
          className="MuiBox-root css-12nrmqt"
        >
          <button
            className="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButton-fullWidth MuiButtonBase-root  css-i4zi72-MuiButtonBase-root-MuiButton-root"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            Chat
            <span
              className="MuiButton-endIcon MuiButton-iconSizeLarge css-z4rqyd-MuiButton-endIcon"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="SendIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M2.01 21 23 12 2.01 3 2 10l15 2-15 2z"
                />
              </svg>
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
